src/iuse.cpp

```diff
@@ -6840,6 +6840,28 @@ int iuse::ehandcuffs( player *p, item *it, bool t, const tripoint &pos )
     return it->type->charges_to_use();
 }
 
+int iuse::foodperson( player *, item *it, bool t, const tripoint &pos )
+{
+    if( t ) {
+        if( calendar::once_every( 1_minutes ) ) {
```

Comment:
Messages spamming every minute will be very annoying.

Reply:
Thanks to  #30543 the message will not display every minute